lib/angular/route.js

```diff
@@ -0,0 +1,18 @@
+var CONSTANTS = require('./constants');
+
+angular.module(CONSTANTS.MAP_DIRECTIVE).config(['$stateProvider', function($stateProvider) {
+
+  var mapStateConfig = {
+    url: '/map',
+    data: {
+      columns: 2
+    },
+    views: {
+      content: {
+        template: '<workorder-map workorders="ctrl.workorders" center="ctrl.center" container-selector="#content"></workorder-map>',
```

Comment:
Can you move this template to a html template file and use the `templateProvider` functionality.

Also, we are not passing `workorders="ctrl.workorders" center="ctrl.center"` to the scope of the directive any more.